types/react-native/index.d.ts

```diff
@@ -5827,6 +5921,27 @@ interface PlatformWebStatic extends PlatformStatic {
     OS: 'web';
 }
 
+/**
+ * Select native platform color
+ * The color must match the string that exists on the native platform
+ *
+ * @see https://reactnative.dev/docs/platformcolor#example
+ */
+export function PlatformColor(color: string): string;
```

Comment:
We might be able to add types to the `color` param instead of using a string to specify all the available ones for each platform.

This might get a bit complicated if the values change depending on the OS version so I just went with `string` for the initial version

Reply:
PlatformColor takes multiple arguments as fallbacks: https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/PlatformColorValueTypes.ios.js#L24-L26